actionpack/lib/action_dispatch/http/parameters.rb

```diff
@@ -78,7 +78,7 @@ def path_parameters=(parameters) # :nodoc:
       # Returns a hash with the \parameters used to form the \path of the request.
       # Returned hash keys are symbols:
       #
-      #   {'action' => 'my_action', 'controller' => 'my_controller'}
+      #   {:action => 'my_action', :controller => 'my_controller'}
```

Comment:
Minor style suggestion:
```suggestion
      #   {action: 'my_action', controller: 'my_controller'}
```

Reply:
Yeah, it is nicer. Only reason why I used "=>" approach is because that is how irb/ruby prints hash with symbols :)

Reply:
If we want to mimic IRB output, we should use double quotes:
```suggestion
      #   {:action=>"my_action", :controller=>"my_controller"}
```

Or, if not, we should still use double quotes :stuck_out_tongue_winking_eye::

```suggestion
      #   { action: "my_action", controller: "my_controller" }
```

Reply:
There you go, no mimicking + double quotes 😃